internal/test/listusers/listusers.go

```diff
@@ -0,0 +1,70 @@
+package listuserstest
+
+import (
+	"strings"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/openfga/openfga/pkg/tuple"
+)
+
+type Assertion struct {
+	Request          *TestListUsersRequest
+	ContextualTuples []*openfgav1.TupleKey `json:"contextualTuples"`
+	Context          *structpb.Struct
+	Expectation      []string
+	ErrorCode        int `json:"errorCode"` // If ErrorCode is non-zero then we expect that the ListUsers call failed.
+}
+
+type TestListUsersRequest struct {
+	Object   string
+	Relation string
+	Filters  []*string `json:"filters"`
```

Comment:
Is the idea here to encode a userset relation in the string? Ex: `group#member`?

Reply:
Yes